apps/test/unit/templates/instructions/TopInstructionsTest.jsx

```diff
@@ -1,31 +1,32 @@
 import React from 'react';
 import {shallow} from 'enzyme';
```

Comment:
Same thing here where most of these tests used to live in TopInstructionsCSPTest